Cicd

mtaillefumier opened the pull request:
cscs run

mtaillefumier commented:
cscs run

mtaillefumier commented:
cscs run test

mtaillefumier commented:
cscs-ci run

mtaillefumier commented:
cscs-ci run

mtaillefumier commented:
cscs-ci run

mtaillefumier commented:
cscs-ci run

mtaillefumier commented:
cscs-ci run

mtaillefumier commented:
cscs-ci run

mtaillefumier commented:
cscs-ci run

mtaillefumier commented:
cscs-ci run

mtaillefumier commented:
cscs-ci run ubuntu-openmpi-cpu

mtaillefumier commented:
cscs-ci run

RMeli commented on ci/baseimage.Dockerfile:
```diff
@@ -0,0 +1,62 @@
+RUN echo $(spack find --format='{prefix.lib}' mpich) > /etc/ld.so.conf.d/mpich.conf
+RUN ldconfig
+
+ENV SPEC_OPENBLAS="cp2k@develop%gcc build_type=Release build_system=cmake +fftw +sirius +elpa +libxc +libint +smm=libxsmm +spglib +cosma +pexsi +plumed +libvori +openmp ^openblas+fortrani ^dbcsr@develop"
```
```suggestion
ENV SPEC_OPENBLAS="cp2k@develop%gcc build_type=Release build_system=cmake +fftw +sirius +elpa +libxc +libint +smm=libxsmm +spglib +cosma +pexsi +plumed +libvori +openmp ^openblas+fortran ^dbcsr@develop"
```

Commit:
CSCS CI/CD

nilsholle committed:
Add apptainer images for znver2 and znver3

mkrack committed:
Minor arch file update

Commit:
Enable tests

juerghutter committed:
Move all ScaLapack routines to the fm folder (#2882)

oschuett committed:
Docs: Launch Sphinx-based manual

oschuett committed:
Docs: Add versions page

oschuett committed:
Docker: Restore Fedora test which got lost in 7b909a0

oschuett committed:
Docs: Add favicon

oschuett committed:
Docs: Restore generation of vim plugin

oschuett committed:
offload: Fix build for HIP_VERSION < 5.2

fstein93 committed:
Add default initialization in HFX/WFC types (#2838)

fstein93 committed:
Fix initializers (#2888)

oschuett committed:
Docs: Remove XSLT-based manual

juerghutter committed:
New default settings for regtesting with DEBUG (#2889)

abussy committed:
ADMM| Fix bugs in ADMMP and ADMMS forces/virial

mkrack committed:
Add apptainer definition files for release 2023.2

mkrack committed:
Insert missing word

mkrack committed:
Increase threshold for eigenvector orthonormality check if enabled

Commit:
[hip] bump version to 50300000

Commit:
Pseudopotential for spin-orbit coupling for TDDFPT

mkrack committed:
Fix indentation

mkrack committed:
Minor fixes

fstein93 committed:
Add more initializers (#2896)

mkrack committed:
Update license dump

mkrack committed:
Sort keywords in new manual and "ruff" script

abussy committed:
Higher precision for new ADMM force regtests

oschuett committed:
Docs: Touch up generate_input_reference.py

oschuett committed:
Docker: Fix i-PI test

fstein93 committed:
Bump ELPA to version 2023.05.001

abussy committed:
XAS_TDP| Fix bug leading to crashes when n_ranks >>> nex_atom

oschuett committed:
Docker: Upgrade to Fedora 38

juerghutter committed:
Energy decomposition analysis (#2906)

fstein93 committed:
Fix dashboard (#2904)

This bugfix circumvents bugs of some older versions of gcc. These are related to default initializers of derived types with allocatable components. In order to circumvent that bug, the respective components have to be explicitly initialized with NULL().

mkrack committed:
Update arch files (ELPA 2023.05.001)

mkrack committed:
Prefer static library if available

mkrack committed:
Compile QUIP for OpenMP

oschuett committed:
Docs: Add README

oschuett committed:
Docs: Exclude README.md from Sphinx build

fstein93 committed:
Fix incorrect density gradients from collocation (#2916)

oschuett committed:
Docs: Touch up README

oschuett committed:
Docs: Add link to keywords in TOC

mkrack committed:
Fix typo in variable name

mkrack committed:
Fix keyword list entries with underscores

mkrack committed:
Use year range in copyright

oschuett committed:
Docs: Fix cross-referencing

mkrack committed:
Fix convention: WRITE with hardcoded unit number

mkrack committed:
Fix typo

mkrack committed:
Adjust tolerances

JWilhelm committed:
Initial commit of scalable, low-memory periodic GW

fstein93 committed:
Refactor rs and pw grids (#2927)

- turn many POINTERs to ALLOCATABLEs
- make POINTERs CONTIGUOUS
- remove some calculations of flop counts where they are actually never used

mkrack committed:
Avoid compiler warning

mkrack committed:
Add docker files for CP2K production containers

These docker files create CP2K containers providing CP2k production binaries for
different target CPUs. Their compressed sizes range from 600-900 MiB depending on
the CP2K version and the inclusion of regression test capabilities.

mkrack committed:
Update README.md

JWilhelm committed:
GW: Add default initializers, remove gw_env_create, and fix double free

mkrack committed:
Extend docs for docker

mkrack committed:
Apply make pretty

mkrack committed:
Run checks for new python script

juerghutter committed:
GAPW TDDFPT (#2932)

JWilhelm committed:
Initialize integers

juerghutter committed:
New Benchmark (small/medium) for different types of functionals (#2934)

mkrack committed:
Fix brackets (#2935)

mkrack committed:
Move flag for the selected Intel compiler to compiler command

mkrack committed:
Switch to latest oneAPI HPC Toolkit version 2023.2.1

Commit:
Add ci/cd tests

mtaillefumier commented:
cscs run

mtaillefumier commented:
cscs-ci run

mtaillefumier commented:
cscs run
